source/support.tex

```diff
@@ -530,251 +530,141 @@
+#define @\defnlibxname{cpp_lib_interpolate}@                       201902L // also in \libheader{cmath}, \libheader{numeric}
+#define @\defnlibxname{cpp_lib_invoke}@                            201411L // also in \libheader{functional}
+#define @\defnlibxname{cpp_lib_is_aggregate}@                      201703L // also in \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_is_constant_evaluated}@             201811L // also in \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_is_final}@                          201402L // also in \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_is_invocable}@                      201703L // also in \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_is_layout_compatible}@              201907L // also in \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_is_null_pointer}@                   201309L // also in \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_is_pointer_interconvertible}@       201907L // also in \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_is_swappable}@                      201603L // also in \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_jthread}@                           201907L // also in \libheader{stop_token}, \libheader{thread}
+#define @\defnlibxname{cpp_lib_latch}@                             201907L // also in \libheader{latch}
+#define @\defnlibxname{cpp_lib_launder}@                           201606L // also in \libheader{new}
+#define @\defnlibxname{cpp_lib_list_remove_return_type}@           201806L // also in \libheader{forward_list}, \libheader{list}
+#define @\defnlibxname{cpp_lib_logical_traits}@                    201510L // also in \libheader{type_traits}
+#define @\defnlibxname{cpp_lib_make_from_tuple}@                   201606L // also in \libheader{tuple}
+#define @\defnlibxname{cpp_lib_make_reverse_iterator}@             201402L // also in \libheader{iterator}
+#define @\defnlibxname{cpp_lib_make_unique}@                       201304L // also in \libheader{memory}
+#define @\defnlibxname{cpp_lib_map_try_emplace}@                   201411L // also in \libheader{map}
+#define @\defnlibxname{cpp_lib_math_constants}@                    201907L // also in \libheader{numbers}
+#define @\defnlibxname{cpp_lib_math_special_functions}@            201603L // also in \libheader{cmath}
+#define @\defnlibxname{cpp_lib_memory_resource}@                   201603L // also in \libheader{memory_resource}
+#define @\defnlibxname{cpp_lib_node_extract}@                      201606L
+  // also in \libheader{map}, \libheader{set}, \libheader{unordered_map}, \libheader{unordered_set}
+#define @\defnlibxname{cpp_lib_nonmember_container_access}@        201411L
+  // also in \libheader{iterator}, \libheader{array}, \libheader{deque}, \libheader{forward_list}, \libheader{list}, \libheader{map}, \libheader{regex}, \libheader{set}, \libheader{string},
```

Comment:
Pre-existing: `<iterator>` is out of order here.

Reply:
Are we usually alphabetical here? In any case, let's not try to fix this in this round.

Reply:
Fixed anyway.